controllers/managedocs_controller.go

```diff
@@ -163,6 +166,14 @@ func (r *ManagedOCSReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			},
 		),
 	)
+	ocsInitializationPredicates := builder.WithPredicates(
+		predicate.NewPredicateFuncs(
+			func(meta metav1.Object, _ runtime.Object) bool {
+				name := meta.GetName()
+				return name == ocsInitializationName
+			},
+		),
+	)
```

Comment:
ocs operator creates the OCSInitialization instance called `ocsinit` so the above predicate reacts only to `ocsinit` and why do we react to any OCSInitialization resource?

Reply:
You can argue the same for the storagecluster resource. The fact of the matter is that we do not try to differentiate on OCSInitialization that we should reconcile and ones that we shouldn't. Not like other resources (e.g. secrets) where we have no interest in touching instances that does not relates to the deployer

Reply:
Deployer owns the storagecluster resource. In case of OCSInitialization , the instance(`ocsinit`) is created by OCS operator. The deployment of toolbox depends only by the spec from `ocsinit` instance. So even if there is a case when another instance of OCSInitialization is created why it is actually need to trigger the reconciliation on creation of it?.

Reply:
I have made the changes as discussed.